src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx

```diff
@@ -211,7 +212,7 @@ export class Frame extends React.Component {
 
   renderLeadHint() {
     if (this.leadsToApp() && !this.state.isExpanded) {
-      return <span className="leads-to-app-hint">{'Called from: '}</span>;
+      return <LeadHint className="leads-to-app-hint">{'Called from: '}</LeadHint>;
```

Comment:
If this is now a styled component can we remove the classname?

Reply:
I need to be very careful in this part of the code and this is something I prefer to change later